changelog/1487.bugfix.2.rst

```diff
@@ -0,0 +1 @@
+Changed `~plasmapy.diagnostics.langmuir.get_electron_temperature` to return inverse slope parameter instead of reciprocal of inverse slope parameter in non-bimaxwellian case. Now returns correct eletron temperature in eV instead of reciprocal of correct electron temperature.
```

Comment:
```suggestion
Corrected the improper inversion of the electron temperature for the
non-bimaxwellian case for
:func:`~plasmapy.diagnostics.langmuir.get_electron_temperature`.
The electron temperature, and not the slope, is a fit parameter of the
curve used by 
:func:`~plasmapy.diagnostics.langmuir.get_electron_temperature`,
so there is no need for the inversion.  The returned value is now the
electron temperature and not its reciprocal.
```

Just some rewording.  I'd like to emphasis the curve fit fits a Te and not a slope.